docker-compose-arm64.yml

```diff
@@ -446,7 +446,7 @@ services:
       STAGEGATE_SECRETSTORESETUP_TOKENS_READYPORT: '54322'
       STAGEGATE_WAITFOR_TIMEOUT: 60s
     hostname: edgex-kong
-    image: kong:2.4.1-alpine
+    image: kong:2.5
```

Comment:
@lenny-intel Got the error while pulling kong:2.5 in arm64 environment.
`INFO:EdgeX deployment:no matching manifest for linux/arm64/v8 in the manifest list entries`
Check on docker hub, the kong version is no linux/arm64/v8 image.

Reply:
@hutchic Colin, was there a glitch in the Kong 2.5 release process?

Reply:
Yes there was and it's since been remedied https://github.com/docker-library/official-images/pull/10737

Reply:
@hutchic , Thank You!. Submit new [PR ](https://github.com/edgexfoundry/edgex-compose/pull/174) to upgrade to 2.5. Should pass TAF smoke tests this time with the ARM64 image available... ;-)